README.md

```diff
@@ -1,5 +1,47 @@
-# C project ( C for College)
-This is a feedback system project. Through this project, we are digitalizing the feedbacks given by the students for their teachers and making them anonymous as well as abstract from the teachers.
-There are many drawbacks of the traditional system. Such as, some of the teachers were able to figure out which feedback is given by whom. The evaluation scores had to be fed into the database, which led to wastage of manpower, time and paper. At times it is difficult to understand the writing of the student which makes it troublesome for understanding the suggestion given by the student. 
-So, this project is designed to tackle all these problems efficiently and only admin will be able to see the outcome of the feedback given by students for the respective teachers in a convenient manner.
-But there is also another problem, some students are not regular in colleges but give the feedback due to which it may harm the profile of a good teacher. So, we will also develop our project in such a way that, the feedbacks of such students will not affect the feedback of a teacher.
+## CollegeProject
+
+A project that involves developing a feedback website for college. The project digitalize the feedbacks given by the students for their teachers. There are few drawbacks in traditional system like
+
+- The teachers are able to find out the identity of students who has given them feedback. Hence, leads to unfair consequences.
+- The evaluation score has to be fed into the database manually which takes a lot of time and require manpower.
+- At times, it is difficult to understand the handwriting of some students, which leads to an ambiguity in suggestions given by such students.
+
+The project tackles all these problems efficiently and focuses on building a fair feedback system.
+
+The website is build up using Django framework with `Python 3.6.5` and `Django 2.0.7`
+***
+## Pre-requisite
+
+Before running the project, you need to create a virtual environment so that it doesn't affect your other projects under development.
+
+To create a  virutal environment follow these steps:
+
+    $ cd ~
+    $ pip install pipenv
+    $ git clone https://github.com/GeekyFreelancers/CollegeProject
+    $ cd CollegeProject
+    $ pipenv install python==3.6.5
+    $ pipenv shell
+
+This will create and activate a virtual environment with Python version as specified. Now you need to install the `Django 2.0.7`(current version)
+
+    $ pip install django==2.0.7
+***
+## Run the Project
+
+First you need to populate the database as:
+
+    $ python manage.py migrate
+
+Then you need to run the server as:
+
+    $ python manage.py runserver
+
+Make sure you are in the CollegeProject directory. Go to the localhost and the project would work.
+***
+
+## References
+
+- To know more about Django, refer https://www.djangoproject.com/
+- To know more about Python, refer https://www.python.org/doc/
+
```

Comment:
All other things are ok

Reply:
good to go.

Reply:
@cypher-naman, approve it please by clicking on the approve option.